rfc-004-sink-regime.md

```diff
@@ -0,0 +1,13 @@
+# Keep the kitchen sink clear
+
+## Summary
+
+Keep our dirty crockery and cutlery from the sink such that the sink can be used all day.
+
+## Problem
+
+Currently we leave a lot of our dirty plates, bowls, and cutlery in the kitchen sink. During the day this mounts up and it becomes hard to use the sink for basic tasks such as running a glass of water.
+
+## Proposal
+
+Members of the team MUST thoroughly rinse their cutlery and crockery, before stacking it neatly in the drying rack. This keeps the sink free throughout the day, and ensures the office looks less like a student house.
```

Comment:
This would also mean we don't run out of cups during the day when we're all in the office (as happened yesterday)

Reply:
Could we also encourage mug re-use?

Reply:
Also is it their cutlery, or the cutlery they've used? 🙃